perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -354,7 +384,8 @@ def WriteObjects(storage_schema, bucket, object_prefix, count,
     host_to_connect: An optional endpoint string to connect to.
   """
 
-  payload_bytes, payload_string = GenerateWritePayload(size)
+  payload = GenerateWritePayload(size)
+  handle = cStringIO.StringIO(payload)
```

Comment:
Yes, I confirmed that creating the handle does not copy the payload. I make a 2GB string in a Python process and then made 1000 handles to it. I checked top, and I couldn't see the memory use growing while I was creating the handles.

This handle is only created once, but I do create per-thread handles in `WriteWorker`.